src/utils/spark.py

```diff
@@ -128,6 +156,8 @@ def get_description(self) -> str:
         if self.cur:
             desc.append(f"cur=[{', '.join(self.cur)}]")
 
+        logger.info(f"Table {self.table_name} description: {', '.join(desc)}")
```

Comment:
**[Nitpick, non-blocking]** Sorry for not catching this on my first review! I'd recommend that we delegate responsibility for logging the description to the calling code in the `submit_jobs` function, which is already a function with a lot of side effects. IMO a getter method like this one should be as functional as possible, and its callers should not depend on its side effects.

Reply:
Agreed! I moved the logging for the getters to a dedicated block in d1054f6.